### ISSUE TITLE
A11y_AzureDataStudio(Schema,Dacpac,Backup)_ Backup Database Dialog_Screenreader: List mark up is missing for options present in Back up combo box

### ISSUE THREAD

**Issue by AbhiramDeshmukh** [microsoft/azuredatastudio#6895]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
Application Name: Azure Data Studio
Application Version: 1.10.0-insider (user setup)
Commit: 1ff9a2ec4c7b88be01666bdd470bbbb4ab95716e
Date: 2019-08-09T07:49:00.254Z
VS Code 1.37.0
Electron: 4.2.7
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
 
**Additional Details:**
Tools Name: Accessibility insight tool
MAS References: MAS1.3.1

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Connections from left menu.
4. Expand the server connected in step 2.
5. Right click an on prem database and select backup from Context menu.
6. Launch JAWS.
7. Navigate up to backup type combo box and expand it.
8. Verify that JAWS is announcing or not the options like Full 1 of 3.
 
**Actual:**
JAWS is not announcing the numbering of options present in Back up type combo box.

**Expected:**
JAWS should announce that Full 1 of 3, Differential 2 of 3 like this.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screen reader users will not understand how many options are present in the combo box.

**MAS Reference:**
MAS 1.3.1- https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for reference:**
[6895_A11y_AzureDataStudio(Schema,Dacpac,Backup)_ Backup Database Dialog_Screenreader.pptx](https://github.com/microsoft/azuredatastudio/files/3546437/6895_A11y_AzureDataStudio.Schema.Dacpac.Backup._.Backup.Database.Dialog_Screenreader.pptx)

**Comment by udeeshagautam** [comment]
Validate if this is only jaws or nvda too

**Comment by udeeshagautam** [comment]
NVDA is announcing the options correctly. This seems to be a JAWs only issue
